build.sbt

```diff
@@ -52,7 +52,34 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     libraryDependencies ++= {
       if (tlIsScala3.value) Seq.empty
       else Seq("org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided)
-    }
+    },
+    mimaBinaryIssueFilters ++= Seq(
+      // Removed implicit conversions for monad transformers from Logger - these were intentionally removed
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.Logger.kleisliLogger"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.Logger.eitherTLogger"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.Logger.optionTLogger"),
+      // Removed implicit conversions for monad transformers from LoggerFactory - these were intentionally removed
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.LoggerFactory.kleisliFactory"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.LoggerFactory.eitherTFactory"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.typelevel.log4cats.LoggerFactory.optionTFactory"),
+      // Added new kernel methods - these are new API additions
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("org.typelevel.log4cats.Logger.kernel"),
+      ProblemFilters.exclude[ReversedMissingMethodProblem](
+        "org.typelevel.log4cats.extras.DeferredLogger.kernel"
+      ),
+      ProblemFilters.exclude[ReversedMissingMethodProblem](
+        "org.typelevel.log4cats.extras.DeferredSelfAwareStructuredLogger.kernel"
+      ),
+      ProblemFilters.exclude[ReversedMissingMethodProblem](
+        "org.typelevel.log4cats.extras.DeferredStructuredLogger.kernel"
+      )
```

Comment:
Bumping `tlBaseVersion` will tell MiMa that you're going to do a minor version bump, and these will go away. 

```
ThisBuild / tlBaseVersion := "2.8"
```